src/js/components/tasks/cards/refresh.tsx

```diff
@@ -0,0 +1,64 @@
+import Button from '@salesforce/design-system-react/components/button';
+import Modal from '@salesforce/design-system-react/components/modal';
+import i18n from 'i18next';
+import React from 'react';
+import { Trans } from 'react-i18next';
+
+import { ExternalLink } from '@/components/utils';
+import { getOrgBehindLatestMsg } from '@/utils/helpers';
+
+// todo @refresh org action
+const RefreshOrgModal = ({
+  orgUrl,
+  missingCommits,
+  isOpen,
+  closeRefreshOrgModal,
+}: {
+  orgUrl: string;
+  missingCommits: number;
+  isOpen: boolean;
+  closeRefreshOrgModal: () => void;
+}) => {
+  let heading = i18n.t('Review Org Behind Latest');
+  if (missingCommits > 0) {
+    heading = `${heading}: ${getOrgBehindLatestMsg(missingCommits, true)}`;
+  }
+  return (
+    <Modal
+      isOpen={isOpen}
+      heading={heading}
+      size="small"
+      onRequestClose={closeRefreshOrgModal}
+    >
+      <div className="slds-p-around_large">
+        <Trans i18nKey="refreshReviewOrgMsg">
+          <strong>[Recommended]</strong> This option will re-create your Review
+          Org with the latest changes, allowing you to review the most recent
+          version.
+        </Trans>
+        <Button
+          label={i18n.t('Refresh Review Org')}
+          variant="brand"
+          className="slds-size_full slds-m-top_medium"
+        />
```

Comment:
This can also get the `onClick={doRefreshOrg}` similar to the button in `orgActions`.

I don't think we need to worry about `isRefreshingOrg` or any loading styles here, if we already made sure the modal is never open during refreshing.